scully.docs.config.ts

```diff
@@ -0,0 +1,14 @@
+import { ScullyConfig, setPluginConfig } from "@scullyio/scully";
+import { baseHrefRewrite } from "@scullyio/scully-plugin-base-href-rewrite";
+
+const pkg = require("./package.json");
+const defaultPostRenderers = [baseHrefRewrite];
+setPluginConfig(baseHrefRewrite, { href: pkg.homepage });
+
+export const config: ScullyConfig = {
+    projectRoot: "./projects/docs/src",
+    projectName: "docs",
+    outDir: "./dist/static",
```

Comment:
I noticed that the build is generating the docs into `dist/static`. This mean that the deployment of docs won't succeed.

You need to either change the directory name that Scully outputs to `docs`
```suggestion
    outDir: "./dist/docs",
```

Or change the publish directory in `.github/workflows/gh-pages.yml`. Currently it points to `dist/docs`
```yaml
    - name: GitHub Pages
      uses: crazy-max/ghaction-github-pages@v1
      with:
        build_dir: dist/docs
      env:
        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

Reply:
rename to docs but we still another folder for `ng build` as scully run based on the built folder